### PR TITLE
Prepare main for sniffer invocation

### PR DESCRIPTION
**Pull request by julianoborba**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -7,10 +7,13 @@
...
+    dest='target_url',
     type=str,
     required=True,
-    help='A url string target that going to be analysed'
+    help='the url to be analysed'
 )
 
-argument_parser.parse_args()
+if __name__ == "__main__":
+    print(argument_parser.parse_args().target_url)
```
Comment (shikow) [inline main.py]: 👀

Comment (resilva87) [reply main.py]: :wat:

Comment (julianoborba) [reply main.py]: I was thinking about have an output until the function that will be called isn't present. Too early code?

File: tests/test_main.py [hunk 2]
```diff
@@ -26,17 +26,6 @@ def test_that_should_print_help_description(self):
             self.assertTrue('show this help message and exit' in str(stdout))
             self.assertIsNone(stderr)
 
-    def test_that_should_return_success_when_using_url_param(self):
```
Comment (carloosamarante) [inline tests/test_main.py]: why you remove? You don't want to use url param? I saw on other file, but in my opinion, i prefer the old version

Comment (julianoborba) [reply tests/test_main.py]: I just realized that when put the sniffer function, this test will run the entire sniff. But that will be better covered by an integration test

Comment (carloosamarante) [reply tests/test_main.py]: hmmm, understand :thinking:

**Pull request closed without merging**